bftengine/src/bftengine/PersistentStorageImp.cpp

```diff
@@ -290,11 +293,27 @@ void PersistentStorageImp::setDescriptorOfLastExecution(const DescriptorOfLastEx
   saveDescriptorOfLastExecution(desc);
 }
 
+void PersistentStorageImp::setDescriptorOfLastStableCheckpoint(
+    const DescriptorOfLastStableCheckpoint &stableCheckDesc) {
+  const size_t bufLen = DescriptorOfLastStableCheckpoint::maxSize(numReplicas_);
+  UniquePtrToChar descBuf(new char[bufLen]);
+  char *descBufPtr = descBuf.get();
+  size_t actualSize = 0;
+  stableCheckDesc.serialize(descBufPtr, bufLen, actualSize);
+  ConcordAssert(actualSize != 0);
+  metadataStorage_->writeInBatch(LAST_STABLE_CHECKPOINT_DESC, descBuf.get(), actualSize);
+}
```

Comment:
Hi @HristoStaykov , does it make sense to add unit tests for the new descriptor in [TestSerialization.cpp](https://github.com/vmware/concord-bft/blob/master/bftengine/tests/testSerialization/TestSerialization.cpp)?

Reply:
Yes, definitely. Will add. Thanks!